tests/unit/amo/sagas/test_home.js

```diff
@@ -142,6 +144,50 @@ describe(__filename, () => {
       expect(loadAction).toEqual(expectedLoadAction);
     });
 
+    it('does not fetch featured themes if includeFeaturedThemes is false', async () => {
+      const state = sagaTester.getState();
+
+      const baseArgs = { api: state.api };
+      const baseFilters = {
+        page_size: LANDING_PAGE_ADDON_COUNT,
+      };
+
+      const collections = [];
+
+      const featuredExtensions = createAddonsApiResult([fakeAddon]);
+      mockSearchApi
+        .expects('search')
+        .withArgs({
```

Comment:
Verifying the call parameters is not relevant to this test (unless I'm missing something?). Also, the call parameters are already verified by other tests. Repeating the validation would introduce a maintenance burden so I'd rather see it left out:

```diff
diff --git a/tests/unit/amo/sagas/test_home.js b/tests/unit/amo/sagas/test_home.js
index 1c6600441..45bd7adfd 100644
--- a/tests/unit/amo/sagas/test_home.js
+++ b/tests/unit/amo/sagas/test_home.js
@@ -145,28 +145,11 @@ describe(__filename, () => {
     });
 
     it('does not fetch featured themes if includeFeaturedThemes is false', async () => {
-      const state = sagaTester.getState();
-
-      const baseArgs = { api: state.api };
-      const baseFilters = {
-        page_size: LANDING_PAGE_ADDON_COUNT,
-      };
-
       const collections = [];
 
       const featuredExtensions = createAddonsApiResult([fakeAddon]);
       mockSearchApi
         .expects('search')
-        .withArgs({
-          ...baseArgs,
-          filters: {
-            ...baseFilters,
-            addonType: ADDON_TYPE_EXTENSION,
-            featured: true,
-            sort: SEARCH_SORT_RANDOM,
-          },
-          page: 1,
-        })
         .returns(Promise.resolve(featuredExtensions));
 
       _fetchHomeAddons({
```